### PR TITLE
Minor fixes in comments or code style

### PR DESCRIPTION
**Pull request by glensc**
Continue https://github.com/dokuwiki/dokuwiki/pull/3942

Fixes some issues that I or IDE made while surfing the code.

### CODE REVIEW

File: inc/HTTP/HTTPClient.php [hunk 1]
```diff
@@ -749,7 +749,7 @@ protected function debugText($info, $var=null){
     /**
      * convert given header string to Header array
      *
-     * All Keys are lowercased.
+     * All Keys are in lowercase.
```
Comment (splitbrain) [inline inc/HTTP/HTTPClient.php:752]: this is not correct. The method converts keys into lower case. This meaning is lost by your change.

**Review by splitbrain — COMMENTED** [review]
Looks good to me so far. Since this is still marked as draft, are you still working on it?

File: inc/HTTP/HTTPClient.php (hunk 1 above)
Comment (glensc) [reply inc/HTTP/HTTPClient.php]: Removed 4a333d73cdeafaf0fc87d9b0253c65b550de53a9

**Comment by glensc** [comment]
Removed Draft. I guess this can be merged since you already reviewed the changes.

**Comment by glensc** [comment]
Rebased against c6ca82d48

**Comment by splitbrain** [comment]
I think most of this has been obsoleted by running rector on the code base. Do you want to rebase it or shall we close it?

**Pull request closed without merging**